src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfFeature.java

```diff
@@ -0,0 +1,1320 @@
+package org.broadinstitute.hellbender.utils.codecs.GENCODE;
+
+import htsjdk.samtools.util.Locus;
+import htsjdk.tribble.Feature;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import scala.tools.nsc.Global;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * A GTF Feature represents one row of a GTF File.
```

Comment:
They do get aggregated together.  I was trying to convey that each row's basic information is stored in an individual GencodeGtfFeature (ignoring sub-features).  I'll update the comment.